uprotocol/cloudevent/factory/ucloudevent.py

```diff
@@ -349,9 +357,126 @@ def extract_integer_value_from_attributes(attr_name, ce: CloudEvent) -> int:
     @staticmethod
     def get_event_type(type):
         return {UMessageType.UMESSAGE_TYPE_PUBLISH: "pub.v1", UMessageType.UMESSAGE_TYPE_REQUEST: "req.v1",
-            UMessageType.UMESSAGE_TYPE_RESPONSE: "res.v1"}.get(type, "")
+                UMessageType.UMESSAGE_TYPE_RESPONSE: "res.v1"}.get(type, "")
 
     @staticmethod
     def get_message_type(ce_type):
         return {"pub.v1": UMessageType.UMESSAGE_TYPE_PUBLISH, "req.v1": UMessageType.UMESSAGE_TYPE_REQUEST,
-            "res.v1": UMessageType.UMESSAGE_TYPE_RESPONSE}.get(ce_type, UMessageType.UMESSAGE_TYPE_UNSPECIFIED)
+                "res.v1": UMessageType.UMESSAGE_TYPE_RESPONSE}.get(ce_type, UMessageType.UMESSAGE_TYPE_UNSPECIFIED)
+
+    @staticmethod
+    def get_content_type_from_upayload_format(payload_format: UPayloadFormat):
+        """
+        Retrieves the content type string based on the provided UPayloadFormat enumeration.<br><br>
+        @param payload_format The UPayloadFormat enumeration representing the payload format.
+        @return The corresponding content type string based on the payload format.
+        """
+        return {
+            UPayloadFormat.UPAYLOAD_FORMAT_JSON: "application/json",
```

Comment:
How do you handle UPAYLOAD_FORMAT_UNSPECIFIED?

Reply:
Ideally, for the 'application/x-protobuf' content type, we should return UPAYLOAD_FORMAT_PROTOBUF and for no or blank content type, we should return UPAYLOAD_FORMAT_UNSPECIFIED. Since you asked me to return UPAYLOAD_FORMAT_PROTOBUF for no content type, there was no option left to return UNSPECIFIED. Let me know if you want me to change it

Reply:
ok thanks